lib/pharos/addons/kubernetes_dashboard.rb

```diff
@@ -6,6 +6,19 @@ class KubeDashboard < Pharos::Addon
       name 'kubernetes-dashboard'
       version '1.8.3'
       license 'Apache License 2.0'
+
+      def install
+        super
+
+        logger.info { "~~> kubernetes-dashboard can be accessed via kubectl proxy at http://localhost:8001/ui" }
```

Comment:
Should this be the complete `KUBECONFIG=~/.pharos/#{@master_node.ip} kubectl proxy`?

Reply:
Or should we have some `pharos-cluster kubectl ...` wrapper for that?

Reply:
IMO we should open separate issue for KUBECONFIG/kubectl output hint. It's not very clear currently how to configure kubectl.